src/main/webapp/WEB-INF/views/index.jsp

```diff
@@ -1,9 +1,32 @@
+
 <%@ page contentType="text/html;charset=UTF-8" language="java" %>
 <html>
 <head>
-    <title>Title</title>
+    <title>Taxi service</title>
 </head>
 <body>
-<h1>Hello World</h1>
+<h1>Taxi service</h1>
+
+<p>
```

Comment:
tag <p> need to describe block with text, but if it just link without text you need not to add tag <p>

Reply:
fixed